go.mod

```diff
@@ -70,6 +70,7 @@ require (
 	k8s.io/utils v0.0.0-20200720150651-0bdb4ca86cbc
 	kubevirt.io/client-go v0.0.0-00010101000000-000000000000
 	kubevirt.io/containerized-data-importer v1.25.0
+	kubevirt.io/controller-lifecycle-operator-sdk v0.1.1
```

Comment:
u need it to this PR ?
i saw other PRs had it just because the author env caused it

Reply:
My fault, sometimes I run `go test` directly to build without wait for bazel.